### PR TITLE
WAT 361 selenium plugin

### PR DESCRIPTION
**Pull request by luckyRoGGeR**
Create browser proxy plugin, that creates selenium driver connection

### CODE REVIEW

File: packages/testring-browser-proxy/src/browser-proxy-controller.ts [hunk 1]
```diff
@@ -71,15 +71,23 @@ export class BrowserProxyController extends PluggableModule implements IBrowserP
         this.pendingCommandsPool.clear();
     }
 
-    private onExit = (): void => {
+    private onExit = (code, error): void => {
+        console.log(code, error);
```
Comment (johnthecat) [inline packages/testring-browser-proxy/src/browser-proxy-controller.ts]: Remove console.log
Are you commiting without hooks?

File: packages/testring-browser-proxy/src/index.ts [hunk 2]
```diff
@@ -3,21 +3,22 @@ import { Transport } from '@testring/transport';
 import { fork } from '@testring/child-process';
 
 import { BrowserProxyController } from './browser-proxy-controller';
-import { BrowserProxy } from './browser-proxy';
 
-const WORKER_PATH = path.join(__dirname, './browser-proxy');
+const WORKER_PATH = path.join(__dirname, './browser-proxy/index.js');
```
Comment (johnthecat) [inline packages/testring-browser-proxy/src/index.ts]: remove index.js please, only `./browser-proxy`

File: packages/testring-e2e-test-app/test/selenium/test/getText.spec.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { run } from 'testring';
+
+run(async (context) => {
+    await context.application.url('https://service.ringcentral.com/');
+
+
+    const val = await context.application.getText('loginCredentialTitle');
+
+    if (val === 'Sign In') {
```
Comment (johnthecat) [inline packages/testring-e2e-test-app/test/selenium/test/getText.spec.js]: use chai, please

File: packages/testring-browser-proxy/src/browser-proxy-controller.ts (hunk 1 above)
Comment (luckyRoGGeR) [reply packages/testring-browser-proxy/src/browser-proxy-controller.ts]: No, last commits were with hooks

File: packages/testring-plugin-selenium-driver/src/plugin/index.ts [hunk 4]
```diff
@@ -0,0 +1,484 @@
...
+        this.localSelenium.stdout.on('data', (data) => {
+            loggerClient.debug(data.toString());
+        });
+
+        this.localSelenium.stderr.on('data', (data) => {
+            loggerClient.debug(data.toString());
+        });
+
+    }
+    public async end(applicant: string) {
```
Comment (dividedby0) [inline packages/testring-plugin-selenium-driver/src/plugin/index.ts]: Gets kinda repetitive here

Comment (dividedby0) [reply packages/testring-plugin-selenium-driver/src/plugin/index.ts]: I'd shorten it further and wrap it in async function which itself awaits for ready state and resloves client

**Pull request merged**